recipe/meta.yaml

```diff
@@ -12,29 +12,16 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
   skip: True  # [py2k and py<34]
   script: python setup.py install --single-version-externally-managed --record record.txt
 
 requirements:
   build:
     - python
     - setuptools
-    - pandas
-    - numpy >=1.12
-    - scikit-learn
-    - matplotlib
-    - jupyter
-    - ipywidgets
-    - ipython
   run:
     - python
-    - numpy >=1.12
-    - scikit-learn
-    - matplotlib
-    - jupyter
-    - ipywidgets
-    - ipython
```

Comment:
These are `run` time, so yes. Only the section above (`build`) can be simplified.